OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -81,25 +81,30 @@ internal static void ProcessOrder(OrderManager orderManager, World world, int cl
 					{
 						var client = orderManager.LobbyInfo.ClientWithIndex(clientId);
 
-						if (client != null)
+						if (client == null)
+							break;
+
+						if (world == null)
```

Comment:
Does `world == null` mean the pre-game lobby? Can we get a comment explaining this if so?